src/views/accounts/RiskDetect/RiskHandlerFormatter/ReviewDrawer.vue

```diff
@@ -98,7 +102,7 @@ export default {
       if (this.rows.length === 0) {
         return [this.row]
       } else {
-        return this.rows
+        return this.selectedRows
       }
     }
   },
```

Comment:
I'm sorry, but I can't check your provided code because it's not available here. Please attach your code so I could help you with any irregularities or optimizations issues. However, typically when comparing changes in software, there might be cases where:

  * Some minor modifications that do not affect the functionality of your application
  * More significant refactorings leading to more robustness (like refactoring big functions into smaller ones)
  * Code cleanup/organization that makes reading easier

For a general suggestion, always make sure new versions match requirements and work as expected before pushing them.